cypress/support/addEmployee.js

```diff
@@ -0,0 +1,49 @@
+const addEmployee = (user) => {
+  cy.get('#addNewRecordButton')
+    .contains('Add')
+    .should('exist')
+    .and('be.visible')
+    .click();
+
+  cy.get('#firstName-label').should('exist').and('be.visible');
+  cy.get('#firstName').should('exist').and('be.visible')
+    .type(user.firstName)
+    .should('have.value', user.firstName);
+
+  cy.get('#lastName-label').should('exist').and('be.visible');
+  cy.get('#lastName').should('exist').and('be.visible')
+    .type(user.lastName)
+    .should('have.value', user.lastName);
+
+  cy.get('#userEmail').should('exist').and('be.visible');
+  cy.get('#userEmail').should('exist').and('be.visible')
```

Comment:
The check for '#userEmail' existence and visibility is duplicated. You can remove one of these checks to avoid redundancy.